.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
This file is not needed, see previous comment

Reply:
Updated [here](https://github.com/okfn/docker-ckan/pull/105/commits/3066c7bb545e134d25ee2ae9ebb0d54cf2693b9e)